### PR TITLE
add UI prompts

### PR DESCRIPTION
**Pull request by mosidrum**
I created a form of UI for your school library app

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ class Student < Person
   def initialize(age, _classroom, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission: parent_permission)
   end
-
+  h
```
Comment (thecodechaser) [inline student.rb]: - [ ] Please remove this unnecessary `h` from here, Users will not be able to start the app because of it.

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+      if choices.key?(user_input)
+        choices[user_input].call
+      else
+        puts 'Invalid selection'
+      end
+
+      break if user_input == 7
+    end
+  end
```
Comment (thecodechaser) [inline app.rb]: - [ ] Please move this code to the `main.rb ` file as requested in the project requirements.

image

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+  def create_book
+    print 'Enter the title of the book:'
+    title = gets.chomp
+    print 'Enter name of author'
+    author = gets.chomp
+    book = Book.new(title, author)
+    @books << book
+    puts 'Book created successfully'
+  end
```
Comment (thecodechaser) [inline app.rb]: - [ ] While creating a book, I'm getting the below error, Please make sure to fix this and users should be able to create and display all entities without any error. (Also getting errors while creating student)

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @mosidrum👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Good job with basic UI. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @mosidrum, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Highlights
- Great job, your project is error free :+1: 
- Nice work you meet all the requirements :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**